xclim/core/indicator.py

```diff
@@ -1027,7 +1040,8 @@ def _update_attrs(cls, args, das, attrs, var_id=None, names=None):
         # In the history attr, call signature will be all keywords and might be in a
         # different order than the real function (but order doesn't really matter with keywords).
         kwargs = OrderedDict(**das)
-        for k, v in args.items():
+        filtered_args = [(k, v) for k, v in args.items() if k in cls._all_parameters]
+        for k, v in filtered_args:
```

Comment:
Is this filter there to remove the copy of the PercentileDataArray? If so, why does it need to be removed? If it has the same name in `das` and `args` then it just replaces its copy, so it's a no-op, no?